WeRockIT-ui/README.md

```diff
@@ -37,3 +37,19 @@ yarn run test:unit
 
 ### Customize configuration
 See [Configuration Reference](https://cli.vuejs.org/config/).
+
+### Setup on Windows
+1. Install Chocolatey, a package manager from Windows, using the following directions from `https://chocolatey.org/install`
+
+    Note: Opted to use powershell since it also directly integrates into VSCode
+
+    Note: If does not work as anticipated right after installation, restart the machine
+
+2. Install Yarn, a project dependency manager, using the instructions from `https://yarnpkg.com/lang/en/docs/install/#windows-stable`
+
+    Execute ```choco install yarn``` within a powershell terminal
+
+    Note: This will also install node.js as a yarn dependency
+
+    Note: Restart your computer for VSCode terminal to have yarn in the path to use within VSCode
```

Comment:
Do you really need to restart your system? Shouldn't you be able to just restart the powershell terminal or VScode terminal?

Reply:
I tried to do so and it didn't work. We could remove it though and let the next person who sets it up to see if it is necessary. This is also me trying to recall what I did a week ago and I could be remembering wrong.  I was surprised today that I was able to use it in the Powershell terminal that hooked up to VSCode today when it wasn't working when I first did the installation.

Reply:
I went ahead and removed it.  :D If it ends up not working we can add it back in.